Realm/RLMSyncSubscription.h

```diff
@@ -135,6 +133,23 @@ RLM_HEADER_AUDIT_BEGIN(nullability, sendability)
 - (void)write:(__attribute__((noescape)) void(^)(void))block
    onComplete:(void(^)(NSError * _Nullable))onComplete __attribute__((unavailable("Renamed to -update:onComplete.")));
 
+/**
+ Synchronously performs any transactions (add/remove/update) to the subscription set within the block. The `onComplete` block is executed after waiting for associated data to be downloaded from the server.
+
+ @param block The block containing actions to perform to the subscription set.
+ @param queue The serial queue to deliver notifications to.
+ @param onComplete A block which is called upon synchronization of
+                   subscriptions from the server. The block will be passed `nil`
+                   if the update succeeded, and an error describing the problem
+                   otherwise.
+ */
+ - (void)update:(__attribute__((noescape)) void(^)(void))block
```

Comment:
Exposing something with a completion queue parameter as a swift async function doesn't really make sense. It'll always just immediately hop off that queue and onto a swift async executor.